americangut/make_phyla_plots.py

```diff
@@ -14,6 +13,9 @@
 from operator import itemgetter
 import colorbrewer
 
+
+use('agg')
```

Comment:
It was complaining about the other lines (imports not being at top of file); will the `noqa` directive fix those complaints? Also, does the placement of this call matter with respect to the imports?